iso_week_test.go

```diff
@@ -4,11 +4,17 @@ import (
 	"testing"
 	"time"
 
-	"bitbucket.org/splice/api/models"
+	"bitbucket.org/splice/api/splice/logger"
```

Comment:
Thanks @kytrinyx my failure motivated me to setup Travis CI to catch potential future issues like this one!